src/test/java/edu/hawaii/its/api/groupings/GroupingUpdateOptAttributeResultTest.java

```diff
@@ -0,0 +1,64 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import edu.hawaii.its.api.util.JsonUtil;
+import edu.hawaii.its.api.wrapper.AssignAttributesResults;
+import edu.internet2.middleware.grouperClient.ws.beans.WsAssignAttributeResult;
+import edu.internet2.middleware.grouperClient.ws.beans.WsAssignAttributesResults;
+import edu.internet2.middleware.grouperClient.ws.beans.WsAttributeDefName;
+
+public class GroupingUpdateOptAttributeResultTest {
+
+    private static Properties properties;
+    private GroupingUpdateOptAttributeResult groupingUpdateOptAttributeResult;
+    @BeforeAll
+    public static void beforeAll() throws Exception {
+        Path path = Paths.get("src/test/resources");
+        Path file = path.resolve("grouper.test.properties");
+        properties = new Properties();
+        properties.load(new FileInputStream(file.toFile()));
+    }
+    private String propertyValue(String key) {
+        return properties.getProperty(key);
+    }
+    @Test
+    public void testConstructor() {
+        String json = propertyValue("ws.assign.attributes.results.null.assign.attribute.result");
+        WsAssignAttributesResults wsAssignAttributesResults =
+                JsonUtil.asObject(json, WsAssignAttributesResults.class);
+        assertNotNull(wsAssignAttributesResults, "WsAssignAttributesResults should not be null after JSON parsing");
+        AssignAttributesResults assignAttributesResults = new AssignAttributesResults(wsAssignAttributesResults);
+        GroupingUpdatedAttributeResult groupingUpdatedAttributeResult =
+                new GroupingUpdatedAttributeResult(assignAttributesResults);
+        GroupingPrivilegeResult optInPrivilegeResult = new GroupingPrivilegeResult(new AssignGrouperPrivilegesResult());
+        GroupingPrivilegeResult optOutPrivilegeResult = new GroupingPrivilegeResult(new AssignGrouperPrivilegesResult());
+        groupingUpdateOptAttributeResult = new GroupingUpdateOptAttributeResult(
+                groupingUpdatedAttributeResult, optInPrivilegeResult, optOutPrivilegeResult);
+        assertNotNull(groupingUpdateOptAttributeResult, "GroupingUpdateOptAttributeResult should not be null");
+        assertEquals(groupingUpdatedAttributeResult, groupingUpdateOptAttributeResult.groupingUpdatedAttributeResult);
+    }
+
+    @Test
+    public void accessors() {
+        if (groupingUpdateOptAttributeResult == null) {
+            testConstructor();
+        }
+        assertNotNull(groupingUpdateOptAttributeResult.getOptInPrivilegeResult(), "OptInPrivilegeResult should not be null");
+        assertEquals("name", groupingUpdateOptAttributeResult.getName(), "Expected name to be 'name'");
+        assertEquals("T", groupingUpdateOptAttributeResult.getUpdatedStatus() ? "T" : "F", "Expected Updated Status to be 'T'");
+        assertEquals("T", groupingUpdateOptAttributeResult.getCurrentStatus() ? "T" : "F", "Expected Current Status to be 'T'");
```

Comment:
Instead of checking for the string "T", use assertTrue(groupingUpdateOptAttributeResult.getUpdatedStatus())